src/GeneratorTwigHelper.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the Symfony MakerBundle package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle;
+
+/**
+ * @author Sadicov Vladimir <sadikoff@gmail.com>
+ *
+ * @internal
+ */
+class GeneratorTwigHelper
+{
+    private $baseLayoutExists;
+
+    public function __construct(FileManager $fileManager)
+    {
+        $this->baseLayoutExists = $fileManager->fileExists('templates/base.html.twig');
```

Comment:
This is something you should calculate not in `__construct()` but only when you need it (so, down in `getHeadPrintCode()`). It doesn't really make a difference, but it's a better practice.